modules/special-rules/pages/war-engines-and-blast-markers.adoc

```diff
@@ -1,11 +1,29 @@
 = War engines and Blast markers
 
-Blast markers affect War Engine detachments in much the same way as other units. 
+Blast markers affect War Engine detachments in much the same way as other units.
 However, *all* War Engine detachments have the {stubborn} ability.
 
-Note that for a War Engine detachment has a total initial Damage Capacity value of more than `15`, you may add +1 to the result of the Leadership test.
+You may add +1 to the result of Leadership tests when the total initial Damage Capacity value of the War Engine detachment is `15` or more, instead when it has `15` or more units.
```

Comment:
Rationalising on '15 or more' is good, but the phrasing is tricky for a couple of reasons. One reason is that we don't want to imply that a detachment gets the bonus based on the total initial DC as it was at the start of the game, but rather on the total initial DCs of all units that remain in play at this moment.

I think that this should work:

```suggestion
Where your War Engine detachment currently has units in it whose *initial* Damage Capacity values total 15 or more, you may add +1 to the result of the Leadership test.
```